frontend/src/metabase/ui/components/inputs/Select/Select.styled.tsx

```diff
@@ -126,13 +126,13 @@ export const getSelectItemsOverrides = (
       padding: "0.75rem",
     },
     item: {
-      color: theme.fn.themeColor("text-dark"),
+      color: "var(--mb-color-text-primary)",
```

Comment:
Theme select item. See the hover state on the dropdown item.

![image](https://github.com/user-attachments/assets/2bed4075-8b7a-45f0-82c1-14f3869a8a25)